content/docs/guides/neon-authorize-future.md

```diff
@@ -0,0 +1,53 @@
+---
+title: The future for Neon Authorize
+subtitle: Our vision for what comes next
+enableTableOfContents: true
+---
+
+[Neon Authorize](/docs/guides/neon-authorize) is an exciting feature, and we want to build more on top of it.
+
+Here's what we'd like to work on next — in no particular order (unless stated otherwise). If you see something you like, or something you'd like to see but don't, let us know in [Discord](https://discord.com/channels/1176467419317940276/1176788564890112042)!
```

Comment:
Here are some ideas of things that we could build next – in no particular order...

Reply:
fixed